examples/set_idp_picture_metadata.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Set IdP picture as metadata
+ *
+ * Flow: External Authentication, Trigger: Post authentication
+ *
+ * @param ctx
+ * @param api
+ */
+function set_idp_picture_metadata(ctx, api) {
+  // return if api undefined
+  if (api === undefined) {
+      return;
```

Comment:
For prod use cases, one should do logging or alerting here for traceability. Can you add an inline comment?